clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py

```diff
@@ -0,0 +1,129 @@
+# coding: utf8
+
+"""Deeplearning prepare data - Clinica Command Line Interface.
+This file has been generated automatically by the `clinica generate template`
+command line tool. See here for more details:
+http://clinica.run/doc/InteractingWithClinica/
+"""
+
+
+import clinica.engine as ce
+
+
+class DeepLearningPrepareDataCLI(ce.CmdParser):
+
+    def define_name(self):
+        """Define the sub-command name to run this pipeline."""
+        self._name = 'deeplearning-prepare-data'
+
+    def define_description(self):
+        """Define a description of this pipeline."""
+        self._description = ('Tensor extraction (Pytorch) from T1w images:\n'
```

Comment:
Proposition (to be approved by @nburgos):
```suggestion
        self._description = ('Prepare data generated Clinica for PyTorch with Tensor extraction:\n'
```
(Proposition follows description of `machinelearning-prepare-spatial-svm`: "Prepare input data for SVM with spatial and anatomical regularization")